services/order.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, MovieSession, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                row=ticket["row"],
+                seat=ticket["seat"],
+                movie_session=MovieSession.objects.get(
+                    id=ticket["movie_session"]

```

Comment:
you don't need to get `MovieSession` object here. You can use `movie_session_id`

Reply:
yes, but...
![image](https://user-images.githubusercontent.com/80958128/199246599-a678f8e0-9ad2-44ff-b419-f7b2914db817.png)

Reply:
movie_session_id